docs/pbc/all/warehouse-management-system/202311.0/unified-commerce/fulfillment-app-fulfill-orders.md

```diff
@@ -0,0 +1,39 @@
+
+
+
+
+## Prerequisites
+
+Log into the Fulfillment App. The login details should be provided by your Back Office administrator. A Back Office user can create or convert an existing Back Office user into a warehouse user. For instructions, see [Create users](/docs/pbc/all/user-management/202400.0/base-shop/manage-in-the-back-office/manage-users/create-users.html) or [Edit users](/docs/pbc/all/user-management/202400.0/base-shop/manage-in-the-back-office/manage-users/edit-users.html).
+
+
+## Fulfill an order
+
+1. In the Fulfillment App, next to the warehouse you want to fulfill an order in, click **Select**.
+2. On the **PICK LISTS** page, next to the order you want to fulfill, click **Start picking**.
+![Screenshots with two picklists]
+3. Based on the description of an item in the picklist, find and collect it in the warehouse.    
+4. Based on whether you found the item, do one of the following next to its card in the Fulfillment App:
+  * If you've collected the item, select or enter the number of items you've collected and click **✓ Done**.
+  * If the item wasn't found, select or enter `0` and click **✓ Done**.
+5. In the dialog that appears, to confirm you've collected the item, click **Confirm**.
+    This moves the item from the **Not Picked** tab to either the **Picked** or the **Not Found** tab. You can double-check the results by opening those tabs.
+6. Repeat steps 3-5 until you process all the items by collecting or marking them as not found.
+7. To finish picking, click **Finish picking**.
+  This opens the **PICK LISTS** page. The picklist you've processed is no longer displayed.
+
+
+
+Once the picking process is complete the state in the State Machine is updated to Picking Finished. To set up State Machine configuration, go (here)[https://docs.spryker.com/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-push-notification-feature.html#set-up-configuration].
+
+A subprocess for DummyPayment01 below describes in detail the state transition for the sales order line item:
```

Comment:
Do we even need to describe the following?

Reply:
We can mention at least that its integrated into the state machine as a subprocess in the example DummyPayment01 (for B2C demoshop) and also MarketplacePayment01 (for B2C Marketplace demoshop). For example we describe some already here. https://docs.spryker.com/docs/pbc/all/order-management-system/202307.0/base-shop/manage-in-the-back-office/orders/change-the-state-of-order-items.html#reference-information-changing-the-state-of-order-items

Reply:
Does it make sense to update [change the state of order items](https://docs.spryker.com/docs/pbc/all/order-management-system/202307.0/base-shop/manage-in-the-back-office/orders/change-the-state-of-order-items.html#reference-information-changing-the-state-of-order-items) with the statuses from the fulfillment app state machine?

Reply:
I think we can add those there, yes.

Reply:
It was flagged by QA, please ensure that changes in the MarketplacePayment01 also described in docs.